appveyor.yml

```diff
@@ -35,6 +36,6 @@ deploy_script:
   - echo password=%password%                         >> %USERPROFILE%\\.pypirc
   - set HOME=%USERPROFILE%
   - pip install wheel twine
-  - ps: if($env:APPVEYOR_REPO_TAG -eq $TRUE) { python -W ignore setup.py && twine upload dist/* }
+  - ps: if($env:APPVEYOR_REPO_TAG -eq $TRUE) { python -W ignore setup.py bdist_wheel bdist_egg && twine upload dist/* }
```

Comment:
I'd completely forgotten that the old wineggbuilder used to upload both .exe and .egg files.  Question for the audience: do we still need .exe installers?

My gut feeling is that people who still use zc.buildout will need .egg, and people who have migrated to pip will need .whl, but probably nobody needs .exe.  Unless there are still people who install stuff manually by downloading installers from PyPI and running them?